<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tester</string>
  </property>
  <widget class="QGroupBox" name="groupBright">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>601</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>조명 조절</string>
   </property>
   <widget class="QLabel" name="labelBright">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>밝기</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBrightPct">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>66</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string>0 %</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="brightSlider">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>160</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="brightSetSavedBtn">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>설정값</string>
    </property>
   </widget>
   <widget class="QPushButton" name="brightOffBtn">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>90</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>끄기</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupWindow">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>601</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>채광 조절</string>
   </property>
   <widget class="QLabel" name="labelBlind">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>블라인드</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBlindDeg">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>66</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string>0 Deg</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelWindow">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>창문</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelWindowDeg">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>66</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string>0 Deg</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QDial" name="blindDial">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>81</width>
      <height>91</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>90</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="sliderPosition">
     <number>0</number>
    </property>
    <property name="notchesVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDial" name="windowDial">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>81</width>
      <height>91</height>
     </rect>
    </property>
    <property name="maximum">
     <number>90</number>
    </property>
    <property name="notchesVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="blindSetSavedBtn">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>설정값</string>
    </property>
   </widget>
   <widget class="QPushButton" name="windowConditioningModeBtn">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>150</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>환기 모드</string>
    </property>
   </widget>
   <widget class="QPushButton" name="blindOffBtn">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>90</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>끄기</string>
    </property>
   </widget>
   <widget class="QPushButton" name="windowCloseBtn">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>190</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>닫기</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
